admin/client/views/home.js

```diff
@@ -124,11 +124,12 @@ var HomeView = React.createClass({
 	},
 	renderOrphanedLists () {
 		if (!this.props.orphanedLists.length) return;
+		let sectionLabel = 'Other';
 		return (
 			<div className="dashboard-group">
-				<div className="dashboard-group__heading">
+				<div className="dashboard-group__heading" data-section-label={sectionLabel}>
 					<span className="dashboard-group__heading-icon octicon octicon-database" />
-					Other
+					{sectionLabel}
```

Comment:
because the label "Other" is now used is multiple(2) places...just abstracting to a local var for clarity